packages/design-system/src/hooks/web3Wagmi.tsx

```diff
@@ -43,7 +42,7 @@ function walletClientToSigner(walletClient: WalletClient) {
 }
 
 /** Hook to convert a viem Wallet Client to an ethers.js Signer. */
-export function useEthersSigner({ chainId }: { chainId?: number } = {}) {
+export function useWagmiEthersSigner({ chainId }: { chainId?: number } = {}) {
```

Comment:
Will have to update the design-system and the import for `useWagmiEthersSigner` in web-3-minting